src/components/Auth/AuthCheckProcessor.ts

```diff
@@ -0,0 +1,173 @@
+  ): void => {
+    //
+    // Part 3:  Handles early exit scenarios where there is no FaceScan to handle -- i.e. User Cancellation, Timeouts, etc.
+    //
+    if (
+      sessionResult.status !==
+      FaceTecSDK.FaceTecSessionStatus.SessionCompletedSuccessfully
+    ) {
+      console.info(
+        `Session was not completed successfully, cancelling.  Status: ${sessionResult.status}`
+      )
+      this.latestNetworkRequest.abort()
+      faceScanResultCallback.cancel()
+      return
+    }
+    //
+    // Part 4:  Get essential data off the FaceTecSessionResult
+    //
+    const parameters = {
+      face_scan: sessionResult.faceScan,
+      audit_trail_image: sessionResult.auditTrail[0],
+      low_quality_audit_trail_image: sessionResult.lowQualityAuditTrail[0],
+      metadata: {
+        sdk_source: 'onfido_web_sdk',
+        sdk_version: '1.0',
+        sdk_metadata: {
+          system: {
+            fingerprint: '',
+            model: '',
+            manufacturer: '',
+            brand: '',
+            product: '',
+            hardware: '',
+          },
+        },
+      },
+    }
+
+    //
+    // Part 5:  Make the Networking Call to Your Servers.  Below is just example code, you are free to customize based on how your own API works.
+    //
+    this.latestNetworkRequest = new XMLHttpRequest()
+    this.latestNetworkRequest.open('POST', `${Config.BaseURL}/auth_3d`)
+    this.latestNetworkRequest.setRequestHeader(
+      'Authorization',
+      `Bearer ${this.sdkToken}`
+    )
+    this.latestNetworkRequest.setRequestHeader(
+      'Content-Type',
+      'application/json'
+    )
+
+    this.latestNetworkRequest.onreadystatechange = () => {
+      //
+      // Part 6:  In our Sample, we evaluate a boolean response and treat true as success, false as "User Needs to Retry",
+      // and handle all other non-nominal responses by cancelling out.  You may have different paradigms in your own API and are free to customize based on these.
+      //
+
+      if (this.latestNetworkRequest.readyState === XMLHttpRequest.DONE) {
+        const responseJSON = JSON.parse(this.latestNetworkRequest.responseText)
+        try {
+          const responseObj = responseJSON
+
+          if (responseObj.success === true) {
+            this.success = true
+
+            faceScanResultCallback.succeed()
+            this.nextStep()
+          } else if (responseObj.success === false) {
+            this.success = false
+
+            console.warn('User needs to retry, invoking retry.')
+            faceScanResultCallback.retry()
+          } else {
+            this.success = false
+
+            console.error('Unexpected API response, cancelling out.')
+            faceScanResultCallback.cancel()
+            this.nextStep()
+          }
+
+          this.events?.emit('complete', {
+            type: 'complete',
+            ...responseObj,
+          })
+          FaceTecSDK.unload(() => {})
```

Comment:
Is there any chance that this `unload` method might fail? In which case would be good to log/catch the error if it happens.

Reply:
This unload method needs a bit more digging since it's not doing what we assumed it was meant for. We have gone in contact with facetec to further investigate this. Irregardless of this, the unload function call or lack thereof should not be an issue for this stage on auth.

Reply:
I'm still waiting for a reply back on this from FaceTec. I'd say it's not a blocker for this initial release, but will raise it later:)

Reply:
@kopijunkie I have discussed this unload issue with Lee from FaceTec and it appears that we do not need this function call at all. This function should only be used to reset the resource directories setup, and nothing directly related to the session flow itself. I've removed these, and addressed the issue we had with re-beginning a new session, so the goal we were trying to achieve with this call has been resolved:)